optashift-employee-rostering-gwtui/src/main/java/org/optaplanner/openshift/employeerostering/gwtui/client/common/JQuery.java

```diff
@@ -0,0 +1,17 @@
+package org.optaplanner.openshift.employeerostering.gwtui.client.common;
+
+import elemental2.dom.HTMLElement;
+import jsinterop.annotations.JsMethod;
+import jsinterop.annotations.JsPackage;
+import jsinterop.annotations.JsType;
+
+@JsType(isNative = true, namespace = JsPackage.GLOBAL)
+public class JQuery {
```

Comment:
I would expect that Errai might have support for this already? @mbarkley

Reply:
We have an example like this in the Errai Tutorial, but we don't provide wrappers for any JavaScript libraries in Errai.